CHANGELOG.md

```diff
@@ -34,6 +31,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   when locale was composed of a languageCode identical to countryCode
 - Make generate demo site work when the default language is set to "fr"
 - Add permission checks to course run admin based on their related course page
+- Fix image srcset in organization_detail template
```

Comment:
why not apply the fix to all srcsets? I think we did the same everywhere...

Reply:
You are right. `category_detail.html` is working but we have to check every template using srcset.

Reply:
After some investigation, issue related to `organization_detail.html` seems to come from a copy pasta failed.